packages/sdk/src/create.ts

```diff
@@ -134,8 +144,12 @@ export async function createFullNode(
 ): Promise<FullNode> {
   options = options ?? {};
 
-  if (!options.pubsubTopics) {
-    options.pubsubTopics = [DefaultPubSubTopic];
+  let pubsubTopics: PubsubTopic[];
```

Comment:
I think we should move it into `WakuNode` class and make `pubsubTopics` optional.

Reply:
Of course, what I meant was we should move this condition inside of `WakuNode` class

```ts
  let pubsubTopics: PubsubTopic[];

  if (!options.shardInfo) {
    pubsubTopics = [DefaultPubsubTopic];
  } else {
    pubsubTopics = shardInfoToPubsubTopics(options.shardInfo);
  }
```

since it is duplicated and fallback to a default pubsub topic is responsibility of a root class and not utility